simbad/command_line/__init__.py

```diff
@@ -212,8 +212,8 @@ def _argparse_rot_options(p):
 
 def _argparse_mr_options(p):
     sg = p.add_argument_group('Molecular Replacement specific options')
-    sg.add_argument('-enan', default=False,
-                    help='Check enantiomorphic space groups')
+    sg.add_argument('-sgalternative', default=None,
```

Comment:
Could we add a short version of this. Maybe something like this (note the double-dash)

```python
sg.add_argument("-sga", "--sgalternative", default=None, help='Check alternative space groups < enant | all >')
```
I think it would make the command line interface cleaner. Theoretically, you could just add those options and then change the CCP4i2 GUI. That should be it. All else should work